<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="Salary">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 246, 230);
font: 20pt
 &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>580</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 246, 230);
font: 20pt
 &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>Salary on hand</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>690</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Made by DP3-2 PIKC RVT || 2023</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>495</width>
     <height>539</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 246, 230);
font: 28pt &quot;Rockwell&quot;;</string>
          </property>
          <property name="text">
           <string>Paycheck Calculator</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 246, 230);
font: 14pt &quot;Rockwell&quot;;</string>
          </property>
          <property name="text">
           <string>On hand</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 246, 230);
font: 18pt &quot;Rockwell&quot;;</string>
          </property>
          <property name="text">
           <string>Input the salary you get before tax:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="salarySlider">
          <property name="minimum">
           <number>500</number>
          </property>
          <property name="maximum">
           <number>3000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="salary">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Rockwell&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 246, 230);
font: 14pt &quot;Rockwell&quot;;</string>
          </property>
          <property name="text">
           <string>Number of dependents:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="depBox">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Rockwell&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 246, 230);
font: 14pt &quot;Rockwell&quot;;</string>
          </property>
          <property name="text">
           <string>Salary tax book submitted to the employer:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="taxBox">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Rockwell&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>Yes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>No</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 246, 230);
font: 14pt &quot;Rockwell&quot;;</string>
          </property>
          <property name="text">
           <string>Additional tax reliefs:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="taxRelief">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Rockwell&quot;;</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="calculate">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell&quot;;</string>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="home">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>Back to Home</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
